files/manifests/k8s-controller-manager.yaml

```diff
@@ -32,7 +32,7 @@ spec:
     - --terminated-pod-gc-threshold=10
     - --use-service-account-credentials=true
     - --kubeconfig=/etc/kubernetes/kubeconfig/controller-manager.yaml
-    - --feature-gates=TTLAfterFinished=true
+    - --feature-gates=TTLAfterFinished=true,CSIMigrationAWS={{ .EnableCSIMigrationAWS }},CSIMigrationAWSComplete={{ .EnableCSIMigrationAWSComplete }}
```

Comment:
So I can understand it better, where do we set the value of `EnableCSIMigrationAWSComplete`?

Reply:
Sure. It will be set in the aws-operator, there's a second PR https://github.com/giantswarm/aws-operator/pull/2979/files#diff-86e71e19ce47e86816692e26dcc5f757d6361c59da4c695766d11d6f3309aa91R351

Reply:
And the `EnableCSIMigrationAWSComplete` is set by default to false once the `Params`struct in `k8scloudconfig` has been initiated by `aws-operator`

Reply:
Thank you!